### PR TITLE
Cielo24 Transcriptions Widget

### PR DESCRIPTION
**Pull request by alexhire**

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by kaltura-hooks** [comment]
Hi @alexhire,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>SVSG  / Cielo24 Transcription Widget Example</title>
```
Comment (mikeweaver) [inline kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html]: Is this an example? Seems like we should update the title.

File: kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+<h1>SVSG / Cielo24 Transcription Widget Demo</h1>
+<h2>This page provides example for how to use Cielo24 Transcription Widget plugin. You can view all the configuration options
+    on the <a href="Cielo24Transcriptions.qunit.html" title="chapters integration">widget integration page</a>.</h2>
+
+<div id="content">
+
+    <br/>
+
+    <!--<div id="kaltura_player" style="width:640px;height:360px;" itemprop="video" itemscope itemtype="http://schema.org/VideoObject" >-->
```
Comment (mikeweaver) [inline kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html]: Do not include commented out code.

File: kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            "targetId": "kaltura_player_1415053771",
+            "wid": "_1795831",
+            "uiconf_id": 26459781,
+            "flashvars": {
+                "streamerType": "auto",
+                "cielo24Transcriptions": {
+                    'DynaTransWindowSize' : 400,
+                    'DynaTransWindowTitle' : 'Media Data Player',
+                    'DynaTransClientLogo' : "img/logo-transparent.png",
+                    "onPageJs1" : "http://s3-us-west-2.amazonaws.com/media-data-player/kalturaIntegrated/stable/js/loader.js"
```
Comment (mikeweaver) [inline kWidget/onPagePlugins/cielo24Transcriptions/Cielo24Transcriptions.html]: Can we use a custom domain name here? Configure CDN.

File: kWidget/onPagePlugins/onPagePlugins.manifest.php [hunk 4]
```diff
@@ -206,12 +206,12 @@
 				'type' => 'string'
 			),
 			'tags' => array(
-				'doc' => 'Default: "chaptering"� Providing a tag name will grab only annotations which have that tag.
```
Comment (zoharbabin) [inline kWidget/onPagePlugins/onPagePlugins.manifest.php]: @mikeweaver - lines 209, 214 and 231 looks like a mistakenly committed charcode changes. please clean it up and recommit.

Comment (mikeweaver) [inline kWidget/onPagePlugins/onPagePlugins.manifest.php]: Fixed.

**Comment by zoharbabin** [comment]
@itaykinnrot @mdale - Can you please review and merge if all good?
This is to introduce the new cielo24 transcript widget into the Studio and Player.kaltura.com features hub. Please advise if anything needs adjustment or fixing.

Thanks!

**Comment by itaykinnrot** [comment]
@alexhire can you resolve conflicts from master so we can merge?

**Comment by alexhire** [comment]
@itaykinnrot I think I can't see conflicts because I don't have write access to your repository.

**Comment by mikeweaver** [comment]
@alexhire just merge the kaltura:master branch into the branch you DO have write access to: SVSG:master. You will be able to resolve the conflicts then.

**Comment by OrenMe** [comment]
closing due to inactive

**Pull request closed without merging**